### PR TITLE
Fix/reorder metrics

### PR DESCRIPTION
**Pull request by john-chrosniak**
# PR Details
## Description

This PR reorders the metrics used for C1T verification testing to match what was finalized in the verification test plan. Additionally, there is a small fix that enables calculating the difference between destinations provided through Rviz and the arrival locations reported by the vehicle.

## Motivation and Context

Having the metrics in order makes data analysis more organized, and the fix for Rviz goals is needed to correctly calculate one of the metrics.

## How Has This Been Tested?

Tested on a bag collected from the turtlebot simulator.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/carma-1-tenth/run_c1t_metric_analysis.py [hunk 1]
```diff
@@ -100,15 +90,20 @@ def test_C1T_07_slowdown_on_turns(self):
...
         # The C1T CMV stops within 0.2 m of the goal destinations
         arrival_distances = check_distance_to_arrival(self.bag_dir)
         self.assertTrue(np.all(arrival_distances < 0.2), "Vehicle did not stop within 0.2 m of goal destination")
 
-    def test_C1T_09_acks_with_MOM(self):
+    def test_C1T_07_acks_with_MOM(self):
         # The C1T CMV sends Mobility Operation message acknowledging when a goal is reached
         self.assertTrue(check_message_published(self.bag_dir, "/outgoing_mobility_operation"), "Message not received on /outgoing_mobility_operation")
 
+    def test_C1T_09_generates_route_from_MOM_timing(self):
```
Comment (willjohnsonk) [inline src/carma-1-tenth/run_c1t_metric_analysis.py:102]: Skipping test 8 atm

File: src/carma-1-tenth/run_c1t_metric_analysis.py [hunk 2]
```diff
@@ -119,7 +114,7 @@ def test_C1T_12_communicate_load_and_unload(self):
         self.assertTrue(check_port_drayage_ack(self.bag_dir, "PICKUP"), "Vehicle did not ack PICKUP correctly")
         self.assertTrue(check_port_drayage_ack(self.bag_dir, "DROPOFF"), "Vehicle did not ack DROPOFF correctly")
 
-    def test_C1T_13_communicate_inspection(self):
+    def test_C1T_16_communicate_inspection(self):
```
Comment (willjohnsonk) [inline src/carma-1-tenth/run_c1t_metric_analysis.py:117]: test 16?

File: src/carma-1-tenth/run_c1t_metric_analysis.py [hunk 3]
```diff
@@ -100,15 +90,20 @@ def test_C1T_07_slowdown_on_turns(self):
         speeds_on_turns = velocities[relevant_velocity_idxs]
         self.assertTrue(np.all(speeds_on_turns > 0.5 * max_target_speed), "Vehicle speed reduced more than 50% on turn")
 
-    def test_C1T_08_stops_close_to_destination(self):
+    def test_C1T_06_stops_close_to_destination(self):
```
Comment (willjohnsonk) [inline src/carma-1-tenth/run_c1t_metric_analysis.py:93]: Didn't we talk about changing this metric?

File: src/carma-1-tenth/run_c1t_metric_analysis.py [hunk 4]
```diff
@@ -100,15 +90,20 @@ def test_C1T_07_slowdown_on_turns(self):
...
 
-    def test_C1T_09_acks_with_MOM(self):
+    def test_C1T_07_acks_with_MOM(self):
         # The C1T CMV sends Mobility Operation message acknowledging when a goal is reached
         self.assertTrue(check_message_published(self.bag_dir, "/outgoing_mobility_operation"), "Message not received on /outgoing_mobility_operation")
 
+    def test_C1T_09_generates_route_from_MOM_timing(self):
+        # The C1T CMV will generate a route within 3 seconds of receiving an incoming mobility operation message
+        time_between_messages = check_message_timing(self.bag_dir, "/incoming_mobility_operation", "/plan")
+        self.assertTrue(np.all(time_between_messages < 3.0), "Time between messages on /incoming_mobility_operation and /plan exceeded 3 seconds")
```
Comment (willjohnsonk) [inline src/carma-1-tenth/run_c1t_metric_analysis.py:105]: Still using 3 seconds, this may be subject to change but might be out of scope for this PR

**Review by willjohnsonk — CHANGES_REQUESTED** [review]

File: src/carma-1-tenth/run_c1t_metric_analysis.py (hunk 1 above)
Comment (john-chrosniak) [reply src/carma-1-tenth/run_c1t_metric_analysis.py:106]: Test 8 added

File: src/carma-1-tenth/run_c1t_metric_analysis.py (hunk 2 above)
Comment (john-chrosniak) [reply src/carma-1-tenth/run_c1t_metric_analysis.py:120]: Tests 10, 11, 14, and 15 are visual observations related to the Port Drayage web UI, so we don't have scripts to compute these

File: src/carma-1-tenth/run_c1t_metric_analysis.py (hunk 3 above)
Comment (john-chrosniak) [reply src/carma-1-tenth/run_c1t_metric_analysis.py:93]: We discussed it but haven't ask for input from PMs or client

**Review by willjohnsonk — APPROVED** [review]

**Pull request merged**